app/views/state_file/questions/submission_confirmation/edit.html.erb

```diff
@@ -38,7 +38,7 @@
 
   <%= render partial: "state_file/questions/submission_confirmation/#{current_state_code}_additional_content" rescue nil %>
 
-  <%= link_to t(".download_state_return_pdf"), StateFile::Questions::SubmissionPdfsController.to_path_helper(action: :show, id: current_intake.latest_submission), class: "button button--primary button--wide" %>
+  <%= link_to t(".download_state_return_pdf"), StateFile::Questions::SubmissionPdfsController.to_path_helper(action: :show, id: current_intake.latest_submission), class: "button button--primary button--wide", role: "button" %>
```

Comment:
[dust] same as other comment, should download PDF be a button or link?

Reply:
Same comment as above from me too